### ISSUE TITLE
[Practice Session] Issue of SonarCloud Organization creation

### ISSUE THREAD

**Issue by kooyunmo** [swsnu/swppfall2019#168]
안녕하세요. [#163](https://github.com/swsnu/swppfall2019/issues/163)에서 공지된 대로 저희 팀 repo에 대해서 SonarCloud organization 생성을 하려했는데 다음과 같이 `This action must be performed by an organization owner`라는 메세지가 뜹니다.

<img width="1269" alt="스크린샷 2019-10-25 오후 4 32 26" src="https://user-images.githubusercontent.com/17061663/67552083-42fa1300-f745-11e9-8062-8986bb1a971e.png">

아무래도 승인 요청이 제대로 보내진 것 같지 않은데 다음과 같이 설정한 후에 update access를 클릭하는 것이 아닌가요?
<img width="555" alt="스크린샷 2019-10-25 오후 4 43 59" src="https://user-images.githubusercontent.com/17061663/67552826-c49e7080-f746-11e9-96fb-a8cb86974bdf.png">

**Comment by kyunggeun-lee** [comment]
네, Request가 제대로 되지 않은 것 같습니다. 제대로 Request가 되려면 아래와 같은 화면이 나왔을 때 "Request" 버튼을 눌러야 합니다.
![image](https://user-images.githubusercontent.com/28984381/67571875-fd9f0b00-f76f-11e9-9103-81c386796107.png)

왜 위와 같은 화면이 나오지 않고 "Update access" 버튼이 나오는지는 저희도 아직 모르겠습니다.
혹시 다른 팀원분의 계정으로 시도해도 같은 현상이 발생하시나요?

**Comment by kyunggeun-lee** [comment]
원인을 발견했습니다. 실수로 학생들이 보낸 SonarCloud install Request 중 하나가 accept 되어서 더이상 Request를 보내지 못하게 된 것 같습니다.
지금 다시 Request 시도해 보시면 잘 되실겁니다.

**Comment by ty8900** [comment]
![image](https://user-images.githubusercontent.com/48665246/67630086-343a6a00-f8c5-11e9-95c2-6a63f561c924.png)
저희 팀 역시 비슷하게 update access로 뜨는데 확인해주실 수 있나요..?

**Comment by GodDrinkTeJAVA** [comment]
Team 17 마찬가지입니다

**Comment by kyunggeun-lee** [comment]
현재는 다음 실습 수업을 위해 SonarCloud에 swsnu organization을 만들어 둔 상태라 더이상 Request를 보낼 수 없는 상태입니다.
대신 숙제는 다음 시간까지 SonarCloud에 가입해 오는 것으로 대체하겠습니다.
